migrations/2022-01-04-034553_add_hidden_column/up.sql

```diff
@@ -0,0 +1,13 @@
+alter table community add column hidden boolean default false;
+
+
+create table mod_hide_community
+(
+    id serial primary key,
+    community_id int references community on update cascade on delete cascade not null,
+    person_id int references person on update cascade on delete cascade not null,
```

Comment:
Not sure how I missed this before. Look at the other mod tables. None of them use `person_id` to refer to a mod. They all use `mod_person_id`. You should be copy-pasting, and tweaking from the existing examples, not building these from scratch and getting all the column names wrong.